core/FamousEngine.js

```diff
@@ -402,12 +405,13 @@ FamousEngine.prototype.addScene = function addScene (scene) {
  * @return {FamousEngine} this
  */
 FamousEngine.prototype.removeScene = function removeScene (scene) {
-    var selector = scene._selector;
+    var selector = scene._id;
```

Comment:
Scene now inherits from Node. This significantly simplifies the Scene constructor.